### DISCUSSION TITLE
IntelliJ plugin

### DISCUSSION THREAD

**Discussion by ghost** [objeck/objeck-lang#156] [Ideas]
Sublime Text and VSCode are advanced code editors. IntelliJ is a real IDE.

https://plugins.jetbrains.com/docs/intellij/custom-language-support.html

**Comment by objeck** [comment]
I happen to use the following _editors_:

1. Sublime
2. Visual Studio
3. Emacs
4. Vi

LSP development was done with Visual Studio Code, as Microsoft created the standard; however, I hardly use it. As an aside, more of the original code for Objeck was written using GNU Emacs on OS X.

**Reply by ghost** [reply]
Ironically Emacs was removed from the list of tested editors for Objeck LSP.

**Comment by ghost** [comment]
As I recall I used to suggest him have a look at NetBeans plugin too (he personally tried Eclipse first but disappointed about the performance). Finally he decided to continue with Sublime Text as it has the best performance. Given he is a GNU Emacs user, I don't think he will like our bloated IDEs :smile:
